targets/csharp/source/PlayFabSDK/source/PlayFabHttp/PlayFabSysHttp.cs

```diff
@@ -43,7 +43,7 @@ public async Task<object> DoPost(string fullUrl, object request, Dictionary<stri
                         // Special case for Authorization header
                         if (headerPair.Key.Equals("Authorization", StringComparison.OrdinalIgnoreCase))
                         {
-                            _client.DefaultRequestHeaders.Authorization = new AuthenticationHeaderValue("Bearer", headerPair.Value);
+                            _client.DefaultRequestHeaders.AadAuthorization = new AuthenticationHeaderValue("Bearer", headerPair.Value);
```

Comment:
We're adding "Bearer" here and elsewhere

Reply:
I'm a little confused as well. Why is this needed here? This, in theory, affects all PlayFab requests (if they have an authorization header), not just those from the editor extension.

Reply:
I'm not sure how this change is needed for unity. I think this was a change from a different branch that got synch'd accidentally. 

Reverting.